src/image-load.cc

```diff
@@ -528,32 +528,8 @@ static void image_loader_area_updated_cb(gpointer,
 	g_mutex_unlock(il->data_mutex);
 }
 
-static void image_loader_area_prepared_cb(gpointer, gpointer data)
```

Comment:
Are we sure it's the proper way fixing GdkPixbufLoader's issue?
Maybe it's enough to remove this handler in ImageLoaderGdk and ImageLoaderSvgz?

Reply:
This callback is still used by jpeg and tiff loaders which do not rely on GdkPixbufLoader.
If we keep it then workaround for svg and xpm could be removed.